src/components/flat-table/flat-table-row/flat-table-row.component.js

```diff
@@ -1,17 +1,43 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Event from '../../../utils/helpers/events';
 import StyledFlatTableRow from './flat-table-row.style';
 
-const FlatTableRow = ({ children }) => {
+const FlatTableRow = ({ children, onClick }) => {
+  let interactiveRowProps = {};
+
+  function onKeyDown(ev) {
```

Comment:
Not a blocking comment, but any reason why `function` and not arrow func?

Reply:
because it should be a function, not a const that is returning a function. In my opinion using an arrow function looks cooler, but it's less readable than the good old `function` keyword.